test/missing.js

```diff
@@ -74,7 +74,9 @@ describe("missing", function() {
 				var callback = function(err, filename) {
 					Array.from(missingDependencies)
 						.sort()
-						.should.containDeep(testCase[2].sort());
+						.should.containDeep(
+							/** @type {Array<string>} */ (testCase[2]).sort()
```

Comment:
You can type `testCases` as `[string, string, string[]][]`

Reply:
done